src/map/mob_spell_container.cpp

```diff
@@ -136,6 +141,11 @@ std::optional<SpellID> CMobSpellContainer::GetSpell()
         }
     }
 
+    // try something really destructive
+    if(HasSevereSpells() && tpzrand::GetRandomNumber(100) < m_PMob->getMobMod(MOBMOD_SEVERE_CHANCE)){
```

Comment:
hmm just realized the whole file has inline bracing before you got here. makes me want to auto-format the whole dang thing...We can do that separately from this pr >.>

Reply:
After dialogue: fixed only for the affected lines by this PR for cleanliness of the PR itself. See ff24165b3b3b2dcc6f2b07f85ffad18954bbdbdb